runtime/plugins/comment/help/comment.md

```diff
@@ -80,18 +80,22 @@ but it is only available for certain filetypes:
 * zsh: `# %s`
 
 If your filetype is not available here, you can simply modify
-the `commenttype` option:
+the `comment.type` option:
 
 ```
-set commenttype "/* %s */"
+set comment.type "/* %s */"
 ```
 
 Or in your `settings.json`:
 
 ```json
 {
     "*.c": {
-        "commenttype": "/* %s */"
+        "comment.type": "/* %s */"
     }
 }
 ```
+
+`commenttype` (without the dot) is the legacy option that is superseded by `comment.type`.
+`commenttype` is still supported but will get deprecated in the future.
+**Use `comment.type` instead.**
```

Comment:
It's better to stick to the rule of wrapping lines to not exceed a certain number of characters per line. Generally in micro's help files this limit is around 80 characters. In this particular file, as I see, it is even less (more like 65 characters), so it's better to stay consistent with that.